.gitignore

```diff
@@ -12,3 +12,5 @@ node_modules
 .vscode
 
 .eslintcache
+
+test/temp/*.css
```

Comment:
Did you use editorconfig?

Reply:
What does that mean? I'm not sure what editorconfig does. Is there something wrong with doing this?

Reply:
I'm not sure but only this file has EoF icon.

Reply:
Oh I see. I saw that too but overlooked it earlier. I think the file just needs another line at the end.